### DISCUSSION TITLE
#Beginner's Guide

### DISCUSSION THREAD

**Discussion by RajivRanasingha** [DuajDiaz/Travel-Website#27] [Q&A] [answered]
How to Split String in Java?

**Comment by Bunny77K9** [comment, marked as answer]
Java API provides several convenient methods to split a string based upon any delimiter e.g. comma, semicolon or colon. You can even use a regular expression to split a big string into several smaller strings.

**Reply by DuajDiaz** [reply]
Thank for your answer!
